android/build.gradle

```diff
@@ -25,10 +25,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
-    // Conditional for compatibility with AGP <4.2.
-    if (project.android.hasProperty("namespace")) {
-        namespace 'dev.google.webcrypto'
-    }
+    namespace 'dev.google.webcrypto'
 
     compileSdkVersion 31
```

Comment:
Is there a reason for this being 31 instead of `flutter.ndkVersion`? (Currently 23.1.7779620)